ArduCopter/Parameters.cpp

```diff
@@ -1027,6 +1027,13 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("PILOT_SPEED_DN", 24, ParametersG2, pilot_speed_dn, 0),
 
+    // @Param: FS_LAND_CONTINUE
+    // @DisplayName: Failsafe land continue
+    // @Description: If FS_LAND_CONTINUE is set for RTL and the copter is in land mode when radio or battery failsafe activates, copter will continue to land. 0 = (default) Disabled (Always take failafe action).  1 = Enabled (Continue with land mode).
```

Comment:
Sorry, but this is wrong now: *If **FS_LAND_CONTINUE** is set for RTL [...]*. That is this new parameter, it can't be set to RTL 😉

Since you have to modify it, please put _Copter_ and _Land_ with capital letter.